xmodule/README.rst

```diff
@@ -24,8 +24,11 @@ For years, edx-platform supported both content frameworks simultaneously via a F
 Fortunately, in 2021, `all remaining XModules were converted into XBlocks`_.
 Since support for the XModule framework is no longer necessary, `the BD-13 project`_ is now working to radically simplify the remaining XBlock runtime implementation.
 
+Recently, this folder was moved from ./common/lib/xmodule/xmodule to its current location, ./xmodule. The reasons that this folder (along with several others) were moved are detailed in `this ADR about the dissolution of sub-projects`_.
```

Comment:
I’d remove dated language that won’t make sense years from now.
```suggestion
This folder was moved from ./common/lib/xmodule/xmodule to its current location, ./xmodule. The reasons that this folder (along with several others) were moved are detailed in `this ADR about the dissolution of sub-projects`_.
```

Reply:
Good point 👍🏻